contracts/ERC721Universal.sol

```diff
@@ -13,69 +16,133 @@ import "./IERC721Universal.sol";
+    }
+
     /**
      * @notice Burns `tokenId`
      * @dev The caller must own `tokenId` or be an approved operator.
      * @param tokenId the id of the token to be burned
      */
-    function burn(uint256 tokenId) public virtual {
-        // Setting an "auth" arguments enables the `_isAuthorized` check which verifies that the token exists
+    function burn(uint256 tokenId) external virtual {
+        // Setting an "auth" argument enables the `_isAuthorized` check which verifies that the token exists
         // (from != 0). Therefore, it is not needed to verify that the return value is not 0 here.
         _update(address(0), tokenId, _msgSender());
-        isBurnedToken[tokenId] = true;
+        isBurned[tokenId] = true;
+    }
+
+    /**
+     * @notice Returns true if the contract implements an interface
+     * @dev Extends the interfaces specified by the standard ERC721
+     *  to additionally respond true when queried about the Id of the
+     *  Universal Minting interface
+     *  Adheres to the ERC165 standard.
+     * @param interfaceId the id of the interface
+     * @return true if this contract implements the interface defined by interfaceId
+     */
+    function supportsInterface(
+        bytes4 interfaceId
+    ) public view virtual override returns (bool) {
+        return
+            interfaceId == type(IERC721UpdatableBaseURI).interfaceId ||
+            interfaceId == type(IERC721Universal).interfaceId ||
+            interfaceId == type(IERC721Broadcast).interfaceId ||
+            super.supportsInterface(interfaceId);
+    }
+
+    /**
+     * @notice Returns the baseURI used to build the tokenURI
+     * @return the baseURI used to build the tokenURI
+     */
+    function baseURI() external view returns (string memory) {
+        return _baseURI();
     }
 
     /**
      * @notice Returns the amount of slots initially owned by an address
      * @dev In the bridgless minting pattern, the correct balance of an owned
-     *  is returned by the separate consensus system, for example, via usage of 
+     *  is returned by the separate consensus system, for example, via usage of
      *  a Universal Node. However, since this method is mandatory in the ERC721 standard,
      *  the only requirement is that the concrete implementation must simply not fail.
      *  The returned value can be an arbitrary constant which should not be used directly
      *  by any other application.
-     * @param owner the address of the owner for which the balance is queried
-     * @return an arbitrary number that should not be used directly. 
+     * @param _owner the address of the owner for which the balance is queried
+     * @return an arbitrary number that should not be used directly.
      */
-    function balanceOf(address owner) public pure override returns (uint256) {
+    function balanceOf(address _owner) public pure override returns (uint256) {
         return 2 ** 96;
```

Comment:
a zero make more sense to me instead of 2 ** 96, it seems less confuse and it will prevent the consumer to try to do something with this number

Reply:
already discussed in the spec, please refer to that issue